esmvaltool/utils/recipe_test_workflow/recipe_test_workflow/meta/rose-meta.conf

```diff
@@ -177,9 +177,22 @@ help=If required, this value must be set in a site-specific configuration file
     =empty string.
 type=quoted
 
+[template variables=RUN_TIME]
+compulsory=true
+description=Time of day at which recipe test workflow will run each day. HH:MM.
```

Comment:
Would it be worth mentioning (maybe in the `help`?) that if the `RUN_TIME` is in the past that the workflow will run immediately?

Reply:
Good idea. Added in [2648ef6](https://github.com/ESMValGroup/ESMValTool/pull/3556/commits/2648ef69407a85e6ce3c9f66a90cb3838cf343ed).